### PR TITLE
Feature/ad auto disable

### PR DESCRIPTION
**Pull request by jflartaud**
Documentation regarding the PR [hashicorp#69](https://github.com/hashicorp/vault-plugin-secrets-ad/pull/69)
Add parameter auto_disable_account in Library Management parameters

### CODE REVIEW

File: website/pages/api-docs/secret/ad/index.mdx [hunk 1]
```diff
@@ -254,6 +254,9 @@ The `library` endpoint configures the sets of service accounts that Vault will o
   automatically checks it back in. Defaults to 24 hours. Setting it to zero reflects an unlimited lending period.
 - `disable_check_in_enforcement` (bool: false, optional): Disable enforcing that service accounts must be
   checked in by the entity or client token that checked them out. Defaults to false.
+- `auto_disable_account` (bool: false, optional): Auto disable account will disable the service accounts on 
+ckeck-in and enable them on check-out. The UserAccountControl attribute is used with the ACCOUNT_DISABLE flag.
```
Comment (taoism4504) [inline website/pages/api-docs/secret/ad/index.mdx:258]:
```suggestion
ckeck-in and enable them on check-out. The UserAccountControl attribute is used with the `ACCOUNT_DISABLE` flag. 
```

**Review by taoism4504 — APPROVED** [review]

**Comment by taoism4504** [comment]
@jflartaud , hi. Can you please fix the conflicts? Thank you.

**Comment by schavis** [comment]
This PR appears abandoned. If there is no activity on the PR by September 8, it will be closed without merging.

**Comment by jflartaud** [comment]
@schavis, hi, as the AD plugin is deprecated, this PR is not useful anymore.

**Pull request closed without merging**